### PR TITLE
Indices Analyze

### PR DESCRIPTION
**Pull request by stevejgordon**
Fix request and response for indices analyze.

### CODE REVIEW

**Review by delvedor — APPROVED** [review]
The changes looks good to me!

@Mpdreamz I'm wondering if we should establish a "style guide" on when and how to merge multiple definitions in a single file. This, for example, seems a good case.

**Comment by Mpdreamz** [comment]
> @Mpdreamz I'm wondering if we should establish a "style guide" on when and how to merge multiple definitions in a single file. This, for example, seems a good case.

++ this seems like a good topic for the next types validation sync :)

**Comment by stevejgordon** [comment]
@Mpdreamz Want me to leave this open for the discussion? Or shall I merge as-is for the time being?

File: output/typescript/types.ts [hunk 1]
```diff
@@ -753,7 +753,8 @@ export interface ICharFilter {
 
 export interface ITokenFilter {
   type: string
-  version: string
+  version?: string
+  stopwords?: Array<string>
```
Comment (Mpdreamz) [inline output/typescript/types.ts:757]: I dont think this should live on `ITokenFilter`. I'm reverting this which might turn indices.analyze red again @stevejgordon

Comment (stevejgordon) [reply output/typescript/types.ts:757]: Ah okay. I'll take another run at it tomorrow.

**Pull request merged**